### ISSUE TITLE
[a11y]: Carbon Tabs replacing the provided ID with an auto generated one. Also always expects a `tabpanel` to be associated so was it mentioned in the aria-controls property.

### ISSUE THREAD

**Issue by prasanthdotv** [carbon-design-system/carbon#17566]
### Package

@carbon/react

### Browser

Chrome

### Operating System

MacOS

### Package version

v1.61

### React version

18.2.0

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checker

### Description

![Screenshot 2024-09-25 at 10 21 12 AM](https://github.com/user-attachments/assets/b8025b16-f7b7-41c7-9d19-3b5e772b83a6)

For the table element the we have given the `aria-describedby`, which is pointing to the tab. Then we are providing the same id in the tab. When it rendered the id is replaced by carbon generated id `ccs-:r1:-tab-0`.

Also this tab element have `aria-controls` which is expecting a tab panel element always. So I had to add dummy visually hidden tab panel with the auto generated id to fix the violation.

### Reproduction/example

https://stackblitz.com/edit/github-dmqw9c?file=src%2FApp.jsx

### Steps to reproduce

1. Create a Tab with a custom id.
2. Inspect the element 
3. Check if the element have the same id.

### Suggested Severity

None

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by prasanthdotv** [comment]
I am also unable to reproduce the issue in stackblitz. But you can see the provided id on tab is not listed anywhere. How can we fix that?

**Comment by tay1orjones** [comment]
Hi @prasanthdotv! 👋 Sorry for the confusion here, Tabs and it's related subcomponents like Tab, TabPanel, etc. must be used when crafting tabs. The internal logic and handling of the components ensures they're fully accessible. 

In this case, the issue can be solved by placing the table inside a TabPanel. By doing so the Tab will be properly associated with the TabPanel and the table will not need an aria-describedby.
